### PR TITLE
Gen3 Monthly Release 2023.02 qa-heal.planx-pla.net 1675107585

### PR DESCRIPTION
**Pull request by PlanXCyborg**
Applying version 2023.02 to qa-heal.planx-pla.net

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# qa-heal.planx-pla.net/manifest.json
## Deployment changes
- dashboard
  - This change will add some CSP and CORS related header to the response that returned from the dashboard services. They should not be interruptive, but each team is encouraged to double check their webpages hosted by dashboard to ensure they still works (https://github.com/uc-cdis/gen3-statics/pull/27)
  - Because of these CSP directives being added, starting from this version, if anyone what to load remote scripts in their dashboard-hosted webpages, they will need to update the CSP directives in this dashboard service if the current directives doesn't fit, or to ship the script files with the page (https://github.com/uc-cdis/gen3-statics/pull/27)
- fence
  - Requires a Fence DB migration (https://github.com/uc-cdis/fence/pull/1068)

**Comment by PlanXCyborg** [comment]
# qa-heal.planx-pla.net/manifest.json
## Deployment changes
- dashboard
  - This change will add some CSP and CORS related header to the response that returned from the dashboard services. They should not be interruptive, but each team is encouraged to double check their webpages hosted by dashboard to ensure they still works (https://github.com/uc-cdis/gen3-statics/pull/27)
  - Because of these CSP directives being added, starting from this version, if anyone what to load remote scripts in their dashboard-hosted webpages, they will need to update the CSP directives in this dashboard service if the current directives doesn't fit, or to ship the script files with the page (https://github.com/uc-cdis/gen3-statics/pull/27)
- fence
  - Requires a Fence DB migration (https://github.com/uc-cdis/fence/pull/1068)

**Comment by PlanXCyborg** [comment]
# qa-heal.planx-pla.net/manifest.json
## Deployment changes
- dashboard
  - This change will add some CSP and CORS related header to the response that returned from the dashboard services. They should not be interruptive, but each team is encouraged to double check their webpages hosted by dashboard to ensure they still works (https://github.com/uc-cdis/gen3-statics/pull/27)
  - Because of these CSP directives being added, starting from this version, if anyone what to load remote scripts in their dashboard-hosted webpages, they will need to update the CSP directives in this dashboard service if the current directives doesn't fit, or to ship the script files with the page (https://github.com/uc-cdis/gen3-statics/pull/27)
- fence
  - Requires a Fence DB migration (https://github.com/uc-cdis/fence/pull/1068)

File: qa-heal.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -8,30 +8,30 @@
...
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
     "frontend-framework": "quay.io/cdis/frontend-framework:0.4.4",
-    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2023.01",
-    "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2023.01",
+    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2023.02",
+    "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2023.02",
     "kayako-wrapper": "quay.io/cdis/kayako-wrapper-service:0.1.1",
-    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2023.01",
+    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2023.02",
     "metadata": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/metadata-service:2.0.1",
```
Comment (mfshao) [inline qa-heal.planx-pla.net/manifest.json:23]: I think this one should be fine to update to 02

File: qa-heal.planx-pla.net/manifest.json [hunk 2]
```diff
@@ -8,30 +8,30 @@
...
-    "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2023.01",
-    "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2023.01",
+    "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2023.02",
+    "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2023.02",
     "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:feat_vlmdSub",
     "requestor": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/requestor:1.7.1",
```
Comment (mfshao) [inline qa-heal.planx-pla.net/manifest.json:27]: and this one

**Comment by PlanXCyborg** [comment]
# qa-heal.planx-pla.net/manifest.json
## Deployment changes
- dashboard
  - This change will add some CSP and CORS related header to the response that returned from the dashboard services. They should not be interruptive, but each team is encouraged to double check their webpages hosted by dashboard to ensure they still works (https://github.com/uc-cdis/gen3-statics/pull/27)
  - Because of these CSP directives being added, starting from this version, if anyone what to load remote scripts in their dashboard-hosted webpages, they will need to update the CSP directives in this dashboard service if the current directives doesn't fit, or to ship the script files with the page (https://github.com/uc-cdis/gen3-statics/pull/27)
- fence
  - Requires a Fence DB migration (https://github.com/uc-cdis/fence/pull/1068)

**Comment by PlanXCyborg** [comment]
# qa-heal.planx-pla.net/manifest.json
## Deployment changes
- dashboard
  - This change will add some CSP and CORS related header to the response that returned from the dashboard services. They should not be interruptive, but each team is encouraged to double check their webpages hosted by dashboard to ensure they still works (https://github.com/uc-cdis/gen3-statics/pull/27)
  - Because of these CSP directives being added, starting from this version, if anyone what to load remote scripts in their dashboard-hosted webpages, they will need to update the CSP directives in this dashboard service if the current directives doesn't fit, or to ship the script files with the page (https://github.com/uc-cdis/gen3-statics/pull/27)
- fence
  - Requires a Fence DB migration (https://github.com/uc-cdis/fence/pull/1068)

**Review by mfshao — APPROVED** [review]

**Pull request merged**